packages/react/src/table/TableScroller.tsx

```diff
@@ -5,24 +5,38 @@ import {
 	useRef,
 	useState,
 } from 'react';
+import { useDraggable } from 'react-use-draggable-scroll';
 import { Box } from '../box';
-import { boxPalette } from '../core';
+import { boxPalette, tokens } from '../core';
 import { Flex } from '../flex';
 import { ScrollbarArrowLeftIcon, ScrollbarArrowRightIcon } from '../icon';
 import { Stack } from '../stack';
+import { ScrollerContext, ScrollerContextType } from './ScrollerContext';
+
+export const SCROLL_OVERLAY_Z_INDEX = 1;
 
 export type TableScrollerProps = { children: ReactNode };
 
+// Frozen columns need a minimum container size to prevent unreacheable/difficult to read table data.
+export const maxContainerBreakpointForFrozenColumns = `@container (max-width: ${
+	tokens.breakpoint.md - 1
+}px)`;
+export const minContainerBreakpointForFrozenColumns = `@container (min-width: ${tokens.breakpoint.md}px)`;
+
 export function TableScroller({ children }: TableScrollerProps) {
 	const trackRef = useRef<HTMLDivElement>(null);
 	const thumbRef = useRef<HTMLButtonElement>(null);
 	const scrollerRef = useRef<HTMLDivElement>(null);
 	const mousePos = useRef({ x: 0, y: 0 });
 
+	const { events } = useDraggable(scrollerRef);
```

Comment:
This is a test based on a question in the design feedback round.

Reply:
Yeah, I wouldn't add this on a whim. Need to also figout how to make text selection not trigger and only use it horizontally.

I would remove since it's just based off 1 piece of feedback that was never considered a part of the component until now

Reply:
It also seemingly prevents vertical scroll when your cursor is over the able now… unless something else has broken that?